api/src/actions/mapLayersActions.js

```diff
@@ -1,25 +1,16 @@
 const model = require('../models/mapLayerModel');
 const { getLayerGeoDataWithUrl } = require('../models/layerGeoDataModel');
 
-const getMapLayersWithGeoData = async () => {
+const getMapLayers = async () => {
```

Comment:
can you add doc comment to the function when you are modifying it?

Reply:
this one function actually still fulfils the previous name, it is returning the mapLayers with link to geojson file from different collection. We should keep the name after all.